### ISSUE TITLE
Interested in using the newer approaches to SQLAlchemy in Pyramid?

### ISSUE THREAD

**Issue by pauleveritt** [cguardia/cookiecutter-pyramid#2]
There's a Pyramid branch with scaffold changes for the newer best practices. I'm up for applying some of them here if you're interested.

**Comment by cguardia** [comment]
Yes, very interested. I just wanted to get this out the door, but there's tons of room for improvement.

**Comment by cguardia** [comment]
Is what you were planning to do for SQLAlchemy template this?

http://git.io/vWsXk

**Comment by pauleveritt** [comment]
I’m stealing those ideas for other purposes, e.g. pyramid_sqltraversal and a cookie cutter template.

—Paul

**Comment by cguardia** [comment]
But will you still apply those ideas here?

**Comment by pauleveritt** [comment]
That’s the thinking.

—Paul

**Comment by cguardia** [comment]
Great, because I think my PR for getting cookiecutter to include virtualenv as a dependency has not a good chance, so the template might be finished in that respect.

**Comment by pauleveritt** [comment]
I’ll confess, I don’t quite understand their reluctance to another dependency. Does that version of virtualenv have known issues on those systems? It’s not like you’re asking to use the system version that comes in Homebrew.

—Paul

**Comment by stevepiercy** [comment]
@pauleveritt @cguardia Epic PR https://github.com/Pylons/pyramid/pull/2024 has been merged.

**Comment by cguardia** [comment]
@pauleveritt any chance you can help me out with this at pycon?

**Comment by pauleveritt** [comment]
Sure, let’s talk to them today.

But I propose we modify our stance. Rather than making them depend on virtualenv, let’s have this option only work for Python3 using the built-in environment support. The command-line option won’t work for Python2 or conda.

—Paul
